content/tracing/span/traceoptions.md

```diff
@@ -0,0 +1,21 @@
+---
+date: 2018-10-25T20:01:43-07:00
+title: "TraceOptions"
+weight: 20
+aliases: [/core-concepts/tracing/span/traceoptions]
+---
+
+### TraceOptions
+TraceOptions is a byte on each OpenCensus span. Currently its last bit is set iff
```

Comment:
iff -> if

Reply:
`iff` means `if and only if` while a normal `if` isn't that strict

Reply:
TIL! Thank you